simtools/applications/make_regular_arrays.py

```diff
@@ -72,101 +68,92 @@ def main():
 
     # Reading site parameters from DB
     db = db_handler.DatabaseHandler(mongo_db_config=db_config)
+    site_pars_db = db.get_site_parameters(
+        site=args_dict["site"], model_version=args_dict["model_version"]
+    )
 
-    site_pars_db = {}
     layout_center_data = {}
+    layout_center_data["center_lat"] = float(site_pars_db["ref_lat"]["Value"]) * u.deg
+    layout_center_data["center_lon"] = float(site_pars_db["ref_long"]["Value"]) * u.deg
+    layout_center_data["center_alt"] = float(site_pars_db["altitude"]["Value"]) * u.m
```

Comment:
I don't remember, but isn't it possible to get the unit from the DB as well? That would be ideal, since the unit might change with time and the code here gets outdated.

Reply:
Yes. This will be fixed in one of the next pull request (already done on a local branch)